presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -65,11 +65,32 @@ Date and Time Functions
 
 .. function:: from_iso8601_timestamp(string) -> timestamp with time zone
 
-    Parses the ISO 8601 formatted ``string`` into a ``timestamp with time zone``.
+    Parses the ISO 8601 formatted date and time ``string``, optionally with
```

Comment:
Remove "time" from "date and time", since it's optional and covered by the second part.

Reply:
done